kafka/pykafka/broker.py

```diff
@@ -77,7 +77,7 @@ def connect(self):
     def fetch_messages(self,
                        partition_requests,
                        timeout=30000,
-                       min_bytes=1024):
+                       min_bytes=1):
```

Comment:
Pretty sure I got 1024 from the Scala driver.  If it only returns 1 byte, that won't even cover the message header, so you couldn't decode anything meaningful anyway.